pkg/languages/defaults/python.go

```diff
@@ -16,8 +17,22 @@ func (p PythonExtractor) ReadDefaults(r reporeader.RepoReader) (map[string]strin
 	if err != nil {
 		return nil, fmt.Errorf("error finding python files: %v", err)
 	}
-	if len(files) > 0 {
-		extractedValues["ENTRYPOINT"] = files[0]
+	for index, file := range files {
+		fileContent, err := r.ReadFile(file)
```

Comment:
i wonder if file is the path or a relative path or what? It's being passed into the Dockerfile so just wondering what kind of path it is

Reply:
This file path is created from the destination directory. So in the integration test, the file path is coming as 'langtest/app.py', which is getting populated in the Dockerfile. I changed it to the base file for now, but might have to make some changes if we're targeting deeper directories other than the root directory.